core/src/crypto/SECP256k1Point.hpp

```diff
@@ -39,21 +39,16 @@ namespace ledger {
     namespace core {
         class SECP256k1Point {
         public:
-            SECP256k1Point(const std::vector<uint8_t>& p);
-            SECP256k1Point operator+(const SECP256k1Point& p) const;
+            explicit SECP256k1Point(const std::vector<uint8_t>& p);
             SECP256k1Point generatorMultiply(const std::vector<uint8_t>& n) const;
-            SECP256k1Point(const SECP256k1Point& p);
+            SECP256k1Point(SECP256k1Point&& p) = default;
```

Comment:
You missed the move assignment operator here - [rule of thee](https://en.cppreference.com/w/cpp/language/rule_of_three)

Reply:
Done. I just remove everything (except ctr) rule of zero :)